src/components/TaskPreviewPane/TaskPreviewPane.tsx

```diff
@@ -31,14 +31,14 @@ export const TaskPreviewPane = () => {
             ? state.calendarReducer.selectedDate
             : defaultSelected,
     );
-    const transformY = useRef(new Animated.Value(-taskPreviewHeight)).current;
+    const transformY = useRef(new Animated.Value(windowHeight)).current;
 
     useEffect(() => {
         //open animation
         Animated.timing(transformY, {
-            toValue: 0,
+            toValue: taskPreviewHeight + 110, //100 seems to work, works on a smaller android phone too
```

Comment:
haha theres a bottom tab navigation that i want it to be above. I'm getting rid of the 110 for now since it looks fine on my phone but when i test on other stuff i think i'll have to find a different solution